mods/ts/rules/trees.yaml

```diff
@@ -19,6 +19,18 @@ BIGBLUE:
 		ResourceType: BlueTiberium
 		Interval: 100
 
+BIGBLUE3:
+	Inherits: ^BlossomTree
+	Tooltip:
+		Name: Large Blue Tiberium Crystal
+	RenderSprites:
+		Palette: bluetiberium
+	RadarColorFromTerrain:
+		Terrain: BlueTiberium
+	SeedsResource:
```

Comment:
It actually is supposed to spawn tib, but at very low rate.

In any case, we can use inheritance here, too.